### ISSUE TITLE
Badges take a relatively long time to be fully loaded

### ISSUE THREAD

**Issue by zjhmale** [cssho/VSMarketplaceBadge#6]
Hi @cssho I just got an issue that the VSMarketplace badges for my VSCode extension can not be displayed on GitHub repo correctly as shown below

![image](https://cloud.githubusercontent.com/assets/6234553/25562230/c5d404e8-2db2-11e7-930c-0d33828008f8.png)
Here is the repo link of my extension: https://github.com/zjhmale/vscode-idris

Days ago I contact to Github support team and here is what they replied to me:

```
Hi there,

I took a look at the three images that are linked there:

https://vsmarketplacebadge.apphb.com/version/zjhmale.Idris.svg
https://vsmarketplacebadge.apphb.com/installs/zjhmale.Idris.svg
https://vsmarketplacebadge.apphb.com/rating/zjhmale.Idris.svg
Our Camo service has a 4 seconds timeout, but all of these images take 5-6 seconds to fully load.

If the underlying service sleeps your project that could be the cause of this, otherwise you may want to reach out to them for more information about why these images are taking so long to load.

I hope this helps!
```

So what actually under the hood will make my badges taking a long time to be fully loaded? I know your service also get information from VSMarketplace, and it will also take more than 10 seconds to open the home page of my extension on VSMarketplace https://marketplace.visualstudio.com/items?itemName=zjhmale.Idris. 

One possible reason in my mind is that I have uploaded too many pictures here https://github.com/zjhmale/vscode-idris/tree/master/images/screenshots, maybe I should move all these features to Github wiki. 

So do you have any ideas to address this, thanks in advance!

**Comment by cssho** [comment]
Thank you for a reporting @zjhmale .
I fixed at df913be9172e73b5fbb26a8588f89db81d3599bb
Data acquisition changed from VSMarket pages to API.

**Comment by zjhmale** [comment]
@cssho Thanks so much for this fix! the badges now can be displayed as expected :)
